src/projectscene/view/trackspanel/trackslistmodel.cpp

```diff
@@ -532,3 +535,33 @@ void TracksListModel::onSelectedTrack(trackedit::TrackId trackId)
         item->setIsSelected(item->trackId() == trackId);
     }
 }
+
+void TracksListModel::onTrackRenameAction(const muse::actions::ActionData& args)
```

Comment:
@Eism i don't like the code duplication here, how can I propagate the action from TrackeditActionsController to QML item?

Reply:
Registering such actions is used in cases where we know for sure that these actions will be used in several places and usually they do not depend on the UI, i.e. in the case of track-rename I would assume that it accepts trackId and newName - the data is prepared and only needs to be applied.

I am not sure that there is an action for renaming a track, so I would suggest not registering such an action and simply handling the rename call - call titleEdit in qml. 
So probably we need a previous solution with `var operations` for MenuButton